battleship-client/src/main/java/com/academy/solid/nie/client/ui/WindowDisplayer.java

```diff
@@ -0,0 +1,57 @@
+package com.academy.solid.nie.client.ui;
+
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+
+/**
+ * Class which purpose is to display window with only one button with provided message on that button
```

Comment:
This class displays a window containing only one button that displays the provided message